src/tmx/TmxUtils/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 PROJECT ( tmxutils CXX )
 FILE (GLOB_RECURSE HEADERS "src/" "*.h*")
 FILE (GLOB_RECURSE SOURCES  "src/" "*.c*")
-
```

Comment:
This is optional but I think it would be helpful if we separate some of this into sub packages like `TmxUtils/src/snmp_client` similar to what we have done with kafka.

Reply:
I would leave this to enhancement later if we would like to reorganize all the files.